CHANGELOG.next.asciidoc

```diff
@@ -74,13 +74,6 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Output errors when Kibana index pattern setup fails. {pull}20121[20121]
 - Fix issue in autodiscover that kept inputs stopped after config updates. {pull}20305[20305]
 - Add service resource in k8s cluster role. {pull}20546[20546]
-- Fixed documentation for commands in beats dev guide {pull}22194[22194]
```

Comment:
This was removed from `CHANGELOG.next.asciidoc` but not added to 7.15.2. I think that's correct as this looks like a docs PR. Just want to make sure though.

Reply:
yes, I removed it manually in https://github.com/elastic/beats/pull/28866/commits/02f28674b712b48c2491e7104d36ed14d1ca0601#diff-cb0601935fcaac3fea29b215282428b964b8b6e1bd5a6b8c68e84d651633c9e3L43

Thanks!